src/app/ad-authoring/ad-authoring.state.ts

```diff
@@ -20,13 +20,13 @@ export class AdAuthoringWorkflowStateContainer {
   private state$: BehaviorSubject<
     AdAuthoringWorkflowState
   > = new BehaviorSubject({
-    landingUrl: 'https://www.amp.dev',
+    landingUrl: '',
     landingType: LandingTypeEnum.AMP,
     callToAction: CallToActionEnum.APPLY_NOW,
     fileSrc: 'https://placekitten.com/300/250',
```

Comment:
Does having a default file src trigger a default non-empty value for file when opening the tool?

Reply:
It doesn't trigger a default non-empty value because even though fileSrc is initialized, assetFile is set to null initially.